web/test/pages/profile/profile-owning.test.tsx

```diff
@@ -496,6 +617,141 @@ describe("profile - owning existing business", () => {
+        }),
+        formationData: generateFormationData({
+          formationFormData: generateFormationFormData({
+            ...emptyAddressData,
+          }),
+        }),
+      });
+      renderPage({ business, municipalities: [generateMunicipality({ displayName: "display name" })] });
+
+      selectByText("Address municipality", "display name");
+      expect(screen.getByLabelText("Address municipality")).toHaveValue("display name");
+      fireEvent.blur(screen.getByLabelText("Address municipality"));
+
+      clickSave();
+      const profileAlert = screen.getByTestId("profile-error-alert");
+
+      await waitFor(() => {
+        expect(profileAlert).toBeInTheDocument();
+      });
+      expect(
+        within(profileAlert).getByText(Config.formation.fields.addressZipCode.label)
+      ).toBeInTheDocument();
+      expect(within(profileAlert).getByText(Config.formation.fields.addressLine1.label)).toBeInTheDocument();
+    });
+
+    it("displays alert when max character limit for business line 1 is reached", async () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          industryId: "generic",
+          businessPersona: "OWNING",
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+        }),
+        formationData: generateFormationData({
+          formationFormData: generateFormationFormData({
+            ...emptyAddressData,
+          }),
+        }),
+      });
+      renderPage({ business, municipalities: [generateMunicipality({ displayName: "display name" })] });
+      fillText("Address line1", "a".repeat(BUSINESS_ADDRESS_LINE_1_MAX_CHAR + 1));
+
+      clickSave();
+      const profileAlert = screen.getByTestId("profile-error-alert");
+
+      await waitFor(() => {
+        expect(profileAlert).toBeInTheDocument();
+      });
+      expect(within(profileAlert).getByText(Config.formation.fields.addressLine1.label)).toBeInTheDocument();
+    });
+
+    it("displays alert when max character limit for business line 2 is reached", async () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          industryId: "generic",
+          businessPersona: "OWNING",
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+        }),
+        formationData: generateFormationData({
+          formationFormData: generateFormationFormData({
+            ...emptyAddressData,
+          }),
+        }),
+      });
+      renderPage({ business, municipalities: [generateMunicipality({ displayName: "display name" })] });
+      fillText("Address line2", "a".repeat(BUSINESS_ADDRESS_LINE_2_MAX_CHAR + 1));
+
+      clickSave();
+      const profileAlert = screen.getByTestId("profile-error-alert");
+
+      await waitFor(() => {
+        expect(profileAlert).toBeInTheDocument();
+      });
+      expect(within(profileAlert).getByText(Config.formation.fields.addressLine2.label)).toBeInTheDocument();
```

Comment:
[boulder] Add a test and logic to cover a case where the user provides a value for Address line 2 and nothing else. We don't want to have a partial address that only has address line 2.

EX:
Given I enter something into address line 2 and all other fields are empty, then all other fields should fire their respective partial address error messages.